Extension/src/githubAPI.ts

```diff
@@ -88,11 +95,11 @@ function getArrayOfBuilds(input: any): Build[] {
     }
     // Only return the the most recent release and insider builds.
     for (let i: number = 0, j: number = 0; i < input.length; i++) {
-        if (isBuild(input[i], false)) {
+        if (isBuild(input[i])) {
             builds[j] = input[i];
             j++;
```

Comment:
Instead of tracking the index manually, usually you just to a [`.push`](https://developer.mozilla.org/en-US/docs/web/javascript/reference/global_objects/array/push) into the array in TypeScript.